src/interfaces/jira.ts

```diff
@@ -138,6 +138,7 @@ export interface JiraDeployment {
 		displayName: string;
 		type: string;
 	};
+	associations?: Array<JiraAssociation | JiraCommitAssociation>;
```

Comment:
We prefer to use the `[]` annotation for arrays.  Also, I think this is more readable if we do `JiraAssociation[]` if you follow my comment below.